SSO-Auth/Config/linking.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Polyfill styles that are missing when serving without dashboard -->
+    <link rel="stylesheet" href="emby-restyle.css" />
+    <link id="theme-style" rel="stylesheet" />
+    <!-- Polyfill material icons -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/material-design-icons-iconfont@6.7.0/dist/material-design-icons.min.css"
+      integrity="sha256-YEkUHnLEghQI0YDx3I/r6cCeYYJHQEdrN+FN2PneTsk="
+      crossorigin="anonymous"
+    />
```

Comment:
@9p4 I know you're going to comment on use of external CDN resources, so I'll justify it now - 

Adding material icons adds bloat to what resources we need to host, and these style elements are purely cosmetic, so it isn't the biggest of deals if a user can't fetch them.

Also, I feel secure about supply-chain attacks, since we're using SRI to verify the stylesheet.

Reply:
I'd prefer that we just embed SVGs for that. There are three icons used and none of them are repeated, so it wouldn't make sense to load it externally. You can find the SVGs in the GH repo for material icons: https://github.com/google/material-design-icons
Running the SVGs through Inkscape and exporting them as minified to make it easier to embed as a one-liner in a span of something.

Reply:
you may be right, but given that user plugin-views are likely to be officially supported by jellyfin-web at some point, I'd prefer to keep the markup / devex consistent between this & something served by the actual router / dashboard, which you can see here https://github.com/9p4/jellyfin-plugin-sso/pull/34#issuecomment-1172898694

Maybe a middle ground is to inline the svgs into the css directly per https://stackoverflow.com/a/21626701

How does that sound?

Reply:
Sounds good! I'm mostly trying to avoid external resources because of bloat

Reply:
Resolved by https://github.com/9p4/jellyfin-plugin-sso/pull/34/commits/53a30cfa251239d1d59584e7bcad7156bfde830a , it actually works pretty well - The icons don't follow font size, and the color is hardcoded but I can live with it for now.